tests/entrypoints/openai/test_basic.py

```diff
@@ -35,6 +73,14 @@ async def client(server):
         yield async_client
 
 
+@pytest.mark.parametrize(
```

Comment:
This should run `test_show_version` twice, once with the default args and once with `--disable-frontend-multiprocessing`, to make sure that the server actually starts in that case.